src/Services/EncounterService.php

```diff
@@ -311,6 +313,60 @@ public function insertEncounter($pid, $data)
             return $results;
         }
     }
+
+    public function updateEncounter($pid, $eid, $data)
+    {
+
+            $facilityService = new FacilityService();
+
+            
+            $facilityresult = $facilityService->getById($data["facility_id"]);
+            $facility = $facilityresult['name'];
+       
```

Comment:
generally best to have 1 empty line between blocks of code

Reply:
Done @bradymiller